### ISSUE TITLE
Stop using the TOML package

### ISSUE THREAD

**Issue by jamespfennell** [jamespfennell/transiter#35]
After #33, we only need it use read the Transiter config, and this can be done using the standard library.

**Comment by jamespfennell** [comment]
Can also rename the config file to be transiter.conf

**Comment by jamespfennell** [comment]
Done as part of the "Automatic PyPi and Docker Hub pushes" branch that was merged into master.
